### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by hulchak**
https://hulchak.github.io/layout_moyo-header/
https://hulchak.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,96 @@
...
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#"
+              class="nav__link is-active"
+            >Apple</a>
```
Comment (sTorba24) [inline src/index.html:48]:
```suggestion
            <a
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
```

Comment (sTorba24) [reply src/index.html:48]: fix everywhere please

File: src/style.css [hunk 2]
```diff
@@ -1 +1,69 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
```
Comment (sTorba24) [inline src/style.css:4]: don't reset styles for *, do it directly for tags

File: src/style.css [hunk 3]
```diff
@@ -1 +1,69 @@
...
+
+.nav__list {
+  display: flex;
+  padding: 0;
+}
+
+.nav__list li {
+  margin-right: 20px;
+  list-style-type: none;
+}
```
Comment (sTorba24) [inline src/style.css:35]: don't style tags, set classes for all `<li>` and style it

File: src/style.css [hunk 4]
```diff
@@ -1 +1,69 @@
...
+
+.nav__list li:last-of-type {
+  margin-right: 0;
+}
+
```
Comment (sTorba24) [inline src/style.css:40]: same

File: src/style.css [hunk 5]
```diff
@@ -1 +1,69 @@
...
+  padding: 22.5px 0;
+  color: #000;
+  font-size: 12px;
+  height: 15px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (sTorba24) [inline src/style.css:52]: you created variable but didn't use it, so do it :)

Comment (sTorba24) [reply src/style.css:52]: use everywhere you need it

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! A few fixes need to be done :)

**Review by AlisaVlasova — APPROVED** [review]

**Pull request closed without merging**